Core/Graphics/Source/Device.cpp

```diff
@@ -46,6 +47,11 @@ namespace Babylon::Graphics
         m_impl->Resize(width, height);
     }
 
+    void Device::UpdateMSAA(uint8_t MSAASamples)
```

Comment:
nit: casing is not right. maybe just use `value` here.
```suggestion
    void Device::UpdateMSAA(uint8_t value)
```